tabs/ports.html

```diff
@@ -62,6 +63,9 @@
                 <td class="functionsCell-gps"><select class="gps_baudrate">
                         <!-- list generated here -->
                 </select></td>
+                <td class="functionsCell-peripherals"><select class="blackbox_baudrate">
```

Comment:
Do I understand correctly that it would be using the same baud rate as selected for blackbox?